masschallenge/impact-api
# [AC-5347] - Deploy Modular Architecture Transition

**Changes Introduced in [AC-5347](https://masschallenge.atlassian.net/browse/AC-5347):**

- merged changes from development

**Test:**

- Create a PR from modular-models-epic to development, see that it is un-mergeable.
- Create a PR from AC-5347 to development, see that it's mergeable.
